### PR TITLE
move swapfile to a role and call it in the pipeline before install

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: playbooks/roles/swapfile/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    path: "{{ swapfile_path }}"
+  register: swapfile_check
+- name: create swap file {{ swapfile_path }}
+  command: fallocate -l {{ swapfile_size }} {{ swapfile_path }}
+  when: not swapfile_check.stat.exists
+- name: set permissions on swap file
+  file:
+    path: "{{ swapfile_path }}"
+    mode: 0600
+- name: mkswap {{ swapfile_path }}
```
Comment (ehelms) [inline playbooks/roles/swapfile/tasks/main.yml]: Does this work? I've seen errors, no entry or just the literal words in the past when using this in a name.

File: playbooks/roles/swapfile/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (ehelms) [inline playbooks/roles/swapfile/tasks/main.yml]: Do you mind putting blank lines between each task? I find it helps with readability, and yes its fair to ask for a lint rule around this.

File: playbooks/roles/swapfile/tasks/main.yml (hunk 1 above)
Comment (evgeni) [inline playbooks/roles/swapfile/tasks/main.yml]: Sure. It just does not work with `with_*` loops.

proof:
```
TASK [swapfile : create swap file /swapfile] ***********************************
Thursday 15 June 2017  18:31:12 +0200 (0:00:00.446)       0:00:07.499 ********* 
changed: [localhost]
```

File: playbooks/roles/swapfile/tasks/main.yml (hunk 2 above)
Comment (evgeni) [inline playbooks/roles/swapfile/tasks/main.yml]: done (yes, a lint rule would be good)

**Comment by ehelms** [comment]
Thanks @evgeni !

**Pull request merged**